src/main/command_executor.cpp

```diff
@@ -106,22 +112,40 @@ bool CommandExecutor::doCommandSingleton(Command* cmd)
       d_solver.get(), d_symman.get(), cmd, d_solver->getDriverOptions().out());
 
   cvc5::Result res;
+  bool hasResult = false;
   const CheckSatCommand* cs = dynamic_cast<const CheckSatCommand*>(cmd);
-  if(cs != nullptr) {
+  if (cs != nullptr)
+  {
     d_result = res = cs->getResult();
+    hasResult = true;
   }
   const CheckSatAssumingCommand* csa =
       dynamic_cast<const CheckSatAssumingCommand*>(cmd);
   if (csa != nullptr)
   {
     d_result = res = csa->getResult();
+    hasResult = true;
+  }
+  // since verbosity is cached, we must check if it was changed by a command
+  const SetOptionCommand* cso = dynamic_cast<const SetOptionCommand*>(cmd);
+  if (cso != nullptr)
+  {
+    if (cso->getFlag() == "verbosity")
+    {
+      d_verbosity = d_solver->getOptionInfo("verbosity").intValue();
+    }
```

Comment:
We decided offline that since this is intended for debug purposes, we will convert this into debug print statement.s